### PR TITLE
Fetch: basic syntax tests for Cross-Origin-Resource-Policy

### PR DESCRIPTION
**Pull request by annevk**
Supplements #11171.

### CODE REVIEW

**Comment by annevk** [comment]
@youennf I based these on your tests (and copied your hello.py). If you want I suppose I could add equivalents for same-site. Not sure if we needed that from the get go.

File: fetch/cross-origin-resource-policy/resources/hello.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def main(request, response):
+    headers = [("Cross-Origin-Resource-Policy", request.GET['corp'])]
+    if 'origin' in request.headers:
+        headers.append(('Access-Control-Allow-Origin', request.headers['origin']))
```
Comment (mikewest) [inline fetch/cross-origin-resource-policy/resources/hello.py]: I assume you're going to use this bit in a future test that checks the `no-cors` restrictions?

File: fetch/cross-origin-resource-policy/syntax.any.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// META: script=/common/get-host-info.sub.js
+
+const crossOriginURL = get_host_info().HTTP_REMOTE_ORIGIN + "/fetch/cross-origin-resource-policy/resources/hello.py?corp=";
+
+[
+  "same",
+  "same, same-origin",
+  "SAME-ORIGIN",
+  "Same-Origin",
+  "same-origin, <>"
```
Comment (mikewest) [inline fetch/cross-origin-resource-policy/syntax.any.js]: `same-origin, same-origin` also seems interesting.

File: fetch/cross-origin-resource-policy/syntax.any.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+].forEach(incorrectHeaderValue => {
+  // Note: an incorrect value results in a successful load, so this test is only meaningful in
+  // implementations with support for the header.
+  promise_test(t => {
+    return fetch(crossOriginURL + encodeURIComponent(incorrectHeaderValue), { mode: "no-cors" });
```
Comment (mikewest) [inline fetch/cross-origin-resource-policy/syntax.any.js]: Might as well check `same-site` as well, perhaps by adding `{{hosts[alt][]}}` to `get-host-info.sub.js` with some reasonable name?

**Review by mikewest — APPROVED** [review]
LGTM!

File: fetch/cross-origin-resource-policy/resources/hello.py (hunk 1 above)
Comment (annevk) [reply fetch/cross-origin-resource-policy/resources/hello.py]: Yeah, this file is from #11171. I copied it here so the tests can run standalone and I don't have to include all prior commits making this harder to review.

**Pull request merged**